### PR TITLE
Add `/chats` command allowing a list of participated chats

### PR DESCRIPTION
**Pull request by anirudhgupta109**
Returns a text file containing name of chats the bot is present in along with the group id

### CODE REVIEW

File: tg_bot/modules/chatlist.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from io import BytesIO
+from typing import Optional, List
+from telegram import TelegramError, Chat, Message
+from telegram import Update, Bot
+from telegram.ext import MessageHandler, Filters, CommandHandler
```
Comment (PaulSonOfLars) [inline tg_bot/modules/chatlist.py]: Imports all over the place, fix them and order them properly

File: tg_bot/modules/chatlist.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from telegram.ext.dispatcher import run_async
+
+import tg_bot.modules.sql.users_sql as sql
+from tg_bot import dispatcher, OWNER_ID, LOGGER
+from tg_bot.modules.helper_funcs.filters import CustomFilters
+
+USERS_GROUP=4
```
Comment (PaulSonOfLars) [inline tg_bot/modules/chatlist.py]: Why is this here

File: tg_bot/modules/chatlist.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+@run_async
+def chats(bot: Bot, update: Update):
+    chats = sql.get_all_chats() or []
+
+    chatfile = 'List of chats.\n'
+    for chat in chats:
+        chatfile += "[x] {} - {}\n".format(chat.chat_name, chat.chat_id)
```
Comment (PaulSonOfLars) [inline tg_bot/modules/chatlist.py]: whats with this formatting, why is there [x]? Its there to mark the user is gbanned in gbanlist, but why here

File: tg_bot/modules/chatlist.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        output.name = "chatlist.txt"
+        update.effective_message.reply_document(document=output, filename="chatlist.txt",
+                                                caption="Here is the list of chats in my database.")
+
+__help__ = ""  # no help string
+
+__mod_name__ = "chatlist"
+
+
+CHATSS_HANDLER = CommandHandler("chats", chats, filters=CustomFilters.sudo_filter)
```
Comment (PaulSonOfLars) [inline tg_bot/modules/chatlist.py]: change to CHATLIST_HANDLER

**Comment by PaulSonOfLars** [comment]
@anirudhgupta109 Happy with the changes?

**Comment by anirudhgupta109** [comment]
Looks good!

**Review by Baoadam — COMMENTED** [review]
Xa
